docs/content/spec/auth/jwt.md

```diff
@@ -63,14 +63,46 @@ Token has 3 main parts:
 
 1.  Headers
 
-    The header of a JSON Web Token is a standard JOSE header. The "typ" field
-    will be "JWT" and it will also contain the "alg" which identifies the
-    signing algorithm used to produce the signature. It also must have a "kid"
-    field, representing the ID of the key which was used to sign the token.
+    The header of a JSON Web Token is a standard JOSE header compliant with
+    [Section 5 of RFC7519](https://datatracker.ietf.org/doc/html/rfc7515#section-5).
 
-    It specifies that this object is going to be a JSON Web token signed using
-    the key with the given ID using the Elliptic Curve signature algorithm
-    using a SHA256 hash.
+    It **must** have:
+    <dl>
+        <dt>
+            <code>alg</code> (Algorithm)
+        </dt>
+        <dd>
+            Identifies the signing algorithm used to produce the signature.
+        </dd>
+        <dt>
+            <code>typ</code> (Type)
+        </dt>
+        <dd>
+            Must be equal to <code>JWT</code> as recommended by [Section 5.1 RFC7519](https://datatracker.ietf.org/doc/html/rfc7519#section-5.1)
+        </dd>
+    </dl>
+
+    It **may** have:
```

Comment:
Should probably say "may have at least one of the following"

Reply:
Or rather "should have at least.."